backend/src/panels/GuidedDevelopmentPanel.ts

```diff
@@ -15,17 +15,45 @@ export class GuidedDevelopmentPanel extends AbstractWebviewPanel {
 	public static GUIDED_DEVELOPMENT = "Guided Development";
 
 	private static channel: vscode.OutputChannel;
+	private uiOptions: any;
 
 	public toggleOutput() {
 		this.outputChannel.showOutput();
 	}
 
+	public loadWebviewPanel(uiOptions?: any) {
+		if (this.webViewPanel && JSON.stringify(uiOptions)===JSON.stringify(this.uiOptions)) {
+			this.webViewPanel.reveal(undefined, true);
+		} else {
+			super.loadWebviewPanel(uiOptions);
+		}
+	}
+
 	public setWebviewPanel(webViewPanel: vscode.WebviewPanel, uiOptions?: any) {
 		super.setWebviewPanel(webViewPanel);
+		if (uiOptions) {
+			this.uiOptions = uiOptions;
+		}
 
 		this.collections = Contributors.getInstance().getCollections();
 
 		this.messages = backendMessages;
+		let renderCollections: IInternalCollection[] = [];
+		if (this.uiOptions && this.uiOptions.renderType) {
+			this.messages = {
+				channelName: "GuidedDev",
+				title: this.uiOptions.title,
+				description: this.uiOptions.description,
+				noResponse: "No response received."
+			};
+			this.collections.forEach(element => {
+				if (this.uiOptions.renderType === "collection" && element.id === this.uiOptions.id) {
+					renderCollections.push(element);
+				}
+			});			
```

Comment:
Perhaps a minor style comment: this line may contain some extra tailing whitespace-charaters (tab)?

Reply:
Thanks. I'll check it and remove the extra tailing whitespace-charaters